.github/workflows/test.yml

```diff
@@ -14,56 +14,17 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - redmine-version: "5.0"
-            ruby-version: "2.7"
-            rdbms: "MySQL"
-          - redmine-version: "5.0"
-            ruby-version: "2.7"
-            rdbms: "PostgreSQL"
-          - redmine-version: "5.0"
-            ruby-version: "3.0"
-            rdbms: "MySQL"
-          - redmine-version: "5.0"
-            ruby-version: "3.0"
-            rdbms: "PostgreSQL"
           - redmine-version: "5.0"
             ruby-version: "3.1"
             rdbms: "MySQL"
           - redmine-version: "5.0"
             ruby-version: "3.1"
             rdbms: "PostgreSQL"
-          - redmine-version: "5.1"
-            ruby-version: "2.7"
-            rdbms: "MySQL"
-          - redmine-version: "5.1"
-            ruby-version: "2.7"
-            rdbms: "PostgreSQL"
-          - redmine-version: "5.1"
-            ruby-version: "3.0"
-            rdbms: "MySQL"
-          - redmine-version: "5.1"
-            ruby-version: "3.0"
-            rdbms: "PostgreSQL"
-          - redmine-version: "5.1"
-            ruby-version: "3.1"
-            rdbms: "MySQL"
-          - redmine-version: "5.1"
-            ruby-version: "3.1"
-            rdbms: "PostgreSQL"
```

Comment:
Removing `3.1` cases are intentional?

Reply:
Yes it is because Ruby 3.1 has already reached EOL and we could use the supported Ruby versions with Redmine 5.1 instead.